owners/bootstrap.js

```diff
@@ -63,7 +63,7 @@ function bootstrap(logger = console) {
     const ownersBot = new OwnersBot(repo);
 
     const initialized = Promise.all([
-      ownersBot.initTeams(github),
+      // ownersBot.initTeams(github),
```

Comment:
?

Reply:
That was temporary development code to prevent hitting the API a lot. Fortunately, tests would have caught it :D